src/Database/DriverFeatureEnum.php

```diff
@@ -47,4 +47,19 @@ enum DriverFeatureEnum: string
      * Window function support (all or partial clauses).
      */
     case WINDOW = 'window';
+
+    /**
+     * Intersect feature support
+     */
+    case INTERSECT = 'intersect';
+
+    /**
+     * Intersect all feature support
+     */
+    case INTERSECT_ALL = 'intersect-all';
+
+    /**
+     * Intersect ordered queries
+     */
+    case INTERSECT_ORDER_BY = 'intersect-order-by';
```

Comment:
@ADmad I have made the changes which lead to remove the SqliteCompiler class because it only had the `_orderedUnion` property. Please take a look and let me know if any other change ir required. Thank you!